cli/src/cluster_query.rs

```diff
@@ -768,6 +770,18 @@ pub fn process_catchup(
             )
         };
 
+        let slot_distance_avg = node_slot as i64 - first_node_slot as i64;
+        let slots_per_second_avg = slot_distance_avg as f64 / f64::from(interval_count * sleep_interval);
+        let time_remaining_avg = (slot_distance as f64 / slots_per_second_avg).round();
+        let time_remaining_avg = if !time_remaining_avg.is_normal() || time_remaining_avg <= 0.0 {
+            "".to_string()
+        } else {
+            format!(
+                "{}",
```

Comment:
Could you move the below `, remaining: ` here not to show only the label like `time_remaining` ?